### PR TITLE
DOCSP-43303-Compass-ToC-refactor

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Shorten and edit the Compass ToC labels with the following guidelines: https://wiki.corp.mongodb.com/pages/viewpage.action?spaceKey=DE&title=FAQ%3A+Table+of+Contents+Labels
 
Official style guidelines for the ToC: Official Style Guidelines for the ToC: https://www.mongodb.com/docs/meta/style-guide/style/toc-labels/

## STAGING

https://deploy-preview-666--docs-compass.netlify.app
(all ToC labels can be seen using the drop down menus)

## JIRA

https://jira.mongodb.org/browse/DOCSP-43303

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66d9f8fcf062608452c7a83c

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by mmaville-mdb — APPROVED** [review]
LGTM! Looks clean!

File: source/create-agg-pipeline.txt [hunk 1]
```diff
@@ -376,13 +376,13 @@ Learn More
...
-   /agg-pipeline-builder/open-saved-pipeline
-   /agg-pipeline-builder/view-pipeline-explain-plan
-   /agg-pipeline-builder/export-pipeline-to-language
-   /agg-pipeline-builder/create-a-view
-   /agg-pipeline-builder/count-pipeline-results
-   /agg-pipeline-builder/pipeline-custom-collation
-   /agg-pipeline-builder/maxtime-ms-pipeline
-   /agg-pipeline-builder/export-pipeline-results
-   /agg-pipeline-builder/aggregation-pipeline-builder-settings
+   Save Pipeline </agg-pipeline-builder/save-agg-pipeline>
```
Comment (mdb-ashley) [inline source/create-agg-pipeline.txt:379]: I think we should leave "Save a Pipeline" and "Open a Pipeline" as-is if we can within the guidelines.

File: source/create-agg-pipeline.txt [hunk 2]
```diff
@@ -376,13 +376,13 @@ Learn More
...
+   Open Pipeline </agg-pipeline-builder/open-saved-pipeline>
+   View Explain Plans </agg-pipeline-builder/view-pipeline-explain-plan>
+   Export to Specific Language </agg-pipeline-builder/export-pipeline-to-language>
```
Comment (mdb-ashley) [inline source/create-agg-pipeline.txt:382]: Perhaps, "Export to a Language"?

File: source/query/filter.txt [hunk 3]
```diff
@@ -486,13 +486,13 @@ How Does the Compass Query Compare to MongoDB and SQL Queries?
...
-   /query-plan
-   /export-query-to-language
-   /query/atlas-search
-   /query/queries
+   Adjust Maximum Time </query/maxtimems>
+   Specify Collation </query/collation>
+   Skip Documents </query/skip>
+   Limit Results </query/limit>
+   View Query Performance </query-plan>
+   Export to Language </export-query-to-language>
```
Comment (mdb-ashley) [inline source/query/filter.txt:496]: As above, perhaps "Export to a Language"?

**Review by mdb-ashley — APPROVED** [review]
LGTM, just left a couple comments for your consideration.

File: source/create-agg-pipeline.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/create-agg-pipeline.txt]: Done!

File: source/create-agg-pipeline.txt (hunk 2 above)
Comment (ltran-mdb2) [reply source/create-agg-pipeline.txt]: Done!

File: source/query/filter.txt (hunk 3 above)
Comment (ltran-mdb2) [reply source/query/filter.txt]: Done!

**Pull request merged**